packages/ecommerce-app-base/.storybook/main.ts

```diff
@@ -7,6 +7,8 @@ const config: StorybookConfig = {
     '@storybook/addon-essentials',
     '@storybook/addon-onboarding',
     '@storybook/addon-interactions',
+    '@storybook/addon-designs',
```

Comment:
I am still not seeing the addon within Storybook itself here. It should show up near the controls, for example:
![Screenshot 2023-08-24 at 3 56 33 PM](https://github.com/contentful/apps/assets/58186851/faafe9c6-ac13-4ece-97ac-b5eba3a25be4)

Reply:
@lilbitner interesting - works on the [preview](https://deploy-preview-4534--ecommerce-app-base-components.netlify.app/) as well 🤔

Reply:
very strange! Then I trust that it works overall!